specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/examples/Settings/UpdateSetting_example.json

```diff
@@ -6,7 +6,7 @@
     "setting": {
       "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/settings/MCAS",
       "name": "MCAS",
-      "kind": "DataExportSetting",
+      "kind": "DataExportSettings",
       "type": "Microsoft.Security/settings",
       "properties": {
```

Comment:
`properties` doesn't exist in the definition and needs to be removed.

Reply:
"properties" exists in the DataExportSettings definition which inherits from Setting.
Technically: we want to pass the derived class (DataExportSettings ) and Setting is our base class. 
Is there a way to overcome this error? Maybe adding it in the definition this way:

    "Setting": {
      "type": "object",
      "description": "Represents a security setting in Azure Security Center.",
      "properties": {},
      "allOf": [
        {
          "$ref": "#/definitions/SettingResource"
        }
      ]
    }

Reply:
yes, leave it in base class.

Reply:
Added the   "properties": {}, to the Setting definition and "additionalProperties": true to the SettingResource
Didn't solve the issue :(

Reply:
Yay fixed it!